### PR TITLE
Add helm chart install 

### PR DESCRIPTION
**Pull request by ashleyschuett**
## What does this PR do?
Adding Helm templates for launching Janus on Kubernetes.

### CODE REVIEW

File: janus/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "janus.fullname" . }}
+  labels:
+{{ include "janus.labels" . | indent 4 }}
```
Comment (harshal-shah) [inline janus/templates/deployment.yaml:6]: super! 👍

File: janus/values.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  - protocol: TCP
+    port: 8081
+    targetPort: 8081
+    name: http-private
+
+ingress:
+  enabled: false
+  name: ops-gateway
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
```
Comment (harshal-shah) [inline janus/values.yaml]: ingress class is handled by the open policy agent already, so this can be omitted.

**Comment by aleksandrzhiliaev** [comment]
+1

Approved with Zappr  Approved with Zappr  Approved with Zappr  Approved with Zappr

**Review by aleksandrzhiliaev — APPROVED** [review]

**Comment by s4nji** [comment]
+1

Approved with Zappr  Approved with Zappr

**Review by s4nji — APPROVED** [review]

**Pull request merged**